detekt-baseline.xml

```diff
@@ -2,6 +2,7 @@
 <SmellBaseline>
   <ManuallySuppressedIssues></ManuallySuppressedIssues>
   <CurrentIssues>
+    <ID>CyclomaticComplexMethod:LicenceStatus.kt$LicenceStatus.Companion$fun lookupLicenceEventByStatus(status: LicenceStatus): LicenceEventType</ID>
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Whereas the other additions to this file are fine, this one should be addressed.